src/scripts/Common/Base.cpp

```diff
@@ -679,6 +679,53 @@ EmoteDesc* MoonScriptCreatureAI::AddEmote(EventType pEventType, const char* pTex
     return NewEmote;
 }
 
+EmoteDesc* MoonScriptCreatureAI::AddEmote(EventType pEventType, uint32 scripttext)
+{
+    EmoteDesc* NewEmote = NULL;
+    NpcScriptText const* ct = sMySQLStore.GetNpcScriptText(scripttext);
+    TextType pType;
+
+    switch (ct->type)
+    {
+    case CHAT_MSG_SAY:
+        pType = Text_Say;
+        break;
+    case CHAT_MSG_MONSTER_YELL:
+        pType = Text_Yell;
+        break;
+    case CHAT_MSG_TEXT_EMOTE:
+        pType = Text_Emote;
+        break;
+    default:
+        LogDebugFlag(LF_SCRIPT_MGR, "MoonScriptCreatureAI::AddEmote() : Invalid Message Type!");
+        break;
+    }
+
+    if (ct != nullptr)
+    {
+        NewEmote = new EmoteDesc(ct->text.c_str(), pType, ct->sound);
+        switch (pEventType)
+        {
+        case Event_OnCombatStart:
```

Comment:
A case label should be indented from its switch statement.
http://www.ascemu.org/wiki/index.php?title=Coding_standards#indent